pages/opening-hours.js

```diff
@@ -44,9 +44,12 @@ const OpeningHours = (props) => (
               error: props.t(props.validatorErrors.opening_hours_monday)
             }}
           >
-            <Heading as="h4" mb={1} size="MEDIUM">
+            <Label
+              class="govuk-label govuk-label—1"
```

Comment:
I think it's supposed to be govuk-label--l not govuk-label--1. As the other examples are label--s and label--m.